akka-docs/src/main/paradox/typed/actor-discovery.md

```diff
@@ -14,29 +14,37 @@ To use Akka Actor Typed, you must add the following dependency in your project:
 
 ## Obtaining Actor references
 
-There are two general ways to obtain @ref:[Actor references](../general/addressing.md#what-is-an-actor-reference): by
-@ref:[creating actors](actor-lifecycle.md#creating-actors) and by discovery using the @ref:[Receptionist](#receptionist).
-
-You can pass actor references between actors as constructor parameters or part of messages.
-
-Sometimes you need something to bootstrap the interaction, for example when actors are running on
-different nodes in the Cluster or when "dependency injection" with constructor parameters is not
-applicable.
+In order for an actor to send a message to one or more other actors it needs to obtain an @ref:[Actor reference](../general/addressing.md#what-is-an-actor-reference). 
+This reference relates to the notion of locality.
+In his 2010 paper, [Actor Model of Computation: Scalable Robust Information Systems](https://arxiv.org/abs/1008.1459), 
+Carl Hewitt relays locality and message processing as, 
+“an Actor can send messages only to addresses for which it has information by the following means: 
+1. that it receives in the message 
+2. that it already had before it received the message 
+3. that it creates while processing the message. “
+
+Similarly, Akka has three general ways to obtain Actor references, but also adds a discovery method
+1. By @ref:[creating actors](actor-lifecycle.md#creating-actors) 
+2. Passed as constructor parameters to an actor on its creation
+3. Passed as parameters of a message received by an actor
+4. By discovery using the @ref:[Receptionist](#receptionist)
 
 ## Receptionist
 
-When an actor needs to be discovered by another actor but you are unable to put a reference to it in an incoming message,
-you can use the `Receptionist`. You register the specific actors that should be discoverable 
-from other nodes in the local `Receptionist` instance. The API of the receptionist is also based on actor messages. 
-This registry of actor references is then automatically distributed to all other nodes in the cluster. 
-You can lookup such actors with the key that was used when they were registered. The reply to such a `Find` request is 
-a `Listing`, which contains a `Set` of actor references that are registered for the key. Note that several actors can be 
+The `Receptionist` is useful when specific actors need to be discoverable by other actors and you need to bootstrap the interaction. 
+For example when they need to be discoverable by actors running either on different nodes in the Cluster, 
```

Comment:
"when they need to be discoverable by" repeated in to sentences in a row, so maybe skip that here?